File: 2502-sort-the-people/sort-the-people.py
class Solution:
    def sortPeople(self, names: List[str], heights: List[int]) -> List[str]:
        N = len(names)
        for i in range(N):
            max_i = i
            for j in range(i+1, N):
                if heights[j] > heights[max_i]:
                    max_i = j
            names[i], names[max_i] = names[max_i], names[i]
            heights[i], heights[max_i] = heights[max_i], heights[i]
        return names

File: 2502-sort-the-people/sort-the-people.go
func sortPeople(names []string, heights []int) []string {
    n :=len(heights)
    for i:=0; i < n; i++{
        swapped :=false
        for j:=0; j < n-1-i; j++{
            if heights[j] < heights[j+1]{
                swapped = true
                heights[j], heights[j+1] = heights[j+1], heights[j]
                names[j], names[j+1] = names[j+1], names[j]

            }
        }
        if !swapped{
            break
        }
    }
    return names
    
}